messages/event_subscription_test.go

```diff
@@ -109,16 +109,16 @@ func TestEventSubscription_EventSupported(t *testing.T) {
 			},
 			false,
 		},
-		{
-			"Lower number of messages",
-			commonDetails,
-			signalDetails{
-				commonDetails.MessageType,
-				commonDetails.View,
-				commonDetails.MinNumMessages - 1,
-			},
-			false,
-		},
+		// {
```

Comment:
commented out code?

Reply:
Removed in f197656a88cc3b013ee8424a61e57dd2735bab8b